Core/include/Acts/EventData/VectorTrackContainer.hpp

```diff
@@ -209,6 +209,10 @@ class ConstVectorTrackContainer final
     return ConstParameters{m_params[itrack].data()};
   }
 
+  ConstCovariance covariance(IndexType itrack) const {
+    return ConstCovariance{m_cov[itrack].data()};
+  }
```

Comment:
unrelated: looking at the pattern `something(Index itrack)` I wondered if it would make sense to have some boxing around this? so it would look like `track(Index itrack).covariance()`

Reply:
That's basically what the `TrackProxy` class does.

```cpp
auto track = trackContainer.getTrack(5);
track.covariance();
```